euphonic/spectra.py

```diff
@@ -718,36 +718,30 @@ def _spectrum_data_name(cls) -> str:
         return f"{cls._spectrum_axis}_data"
 
     @classmethod
-    def _spectrum_raw_data_name(cls) -> str:
+    def _raw_spectrum_data_name(cls) -> str:
```

Comment:
Looks good. At some point I can see these wanting to become `property` s, but not for this PR.

Reply:
Unfortunately it is no longer allowed to make a `property` that is also a `classmethod`.

Reply:
Given an instance has the properties of the class does that come into play? Nevermind then. Regardless, like I say, not for this PR. Let's get this merged when the tests are happy!

Reply:
It seems useful/appropriate that this information can be accessed _before_ constructing the instance, as it is related to the `__init__` signature.

Reply:
It doesn't actually look very hard to make a custom descriptor for similar behaviour. I _assume_ there is a reason it isn't actively encouraged 🤔  (or routinely imported from some library)

https://stackoverflow.com/questions/76378373/what-are-some-alternatives-to-using-classmethod-and-property-decorators-together